docs/configuration-file.md

```diff
@@ -8,6 +8,7 @@ The `generator` property from `package.json` file must contain a JSON object tha
 |Name|Type|Description|
 |---|---|---|
 |`renderer`| String | Its value can be either `react` or `nunjucks` (default).
+|`apiVersion`| String | Determines which **major** version of the [Parser-API](https://github.com/asyncapi/parser-api) the template uses. For example, `v1` for `v1.x.x`. If not specified, the Generator assumes the template is not compatible with the [Parser-API](https://github.com/asyncapi/parser-api) so it will use the [Parser-JS v1 API](https://github.com/asyncapi/parser-js/tree/v1.18.1#api-documentation). If the template uses a version of the [Parser-API](https://github.com/asyncapi/parser-api) that is not supported by the Generator, the Generator will throw an error.
```

Comment:
Good point. I can add that, because I consider is how the generator should behave so it doesn't goes with unexpected behaviors. However, I'm wondering if that will be considered as breaking change: templates that, for some reason, include now the `apiVersion` field (I know it wasn't declared, but imagine someone added it because "whatever") will start receiving those exceptions.
Obviously, this is not gonna happen with the list of official supported templates. 

In my opinion, it is not a BC because `apiVersion` was never supported nor even mentioned in the documentation until this PR. WDYT?

Reply:
I added this change so it throws an exception during parameters validation if the `apiVersion` is present but not valid value (within a range).

See https://github.com/asyncapi/generator/pull/960/commits/dbb27c469f174dff0f42be29b5b8fa22d0b6ac7e

cc @derberg @fmvilas